set.mm

```diff
@@ -99022,6 +99022,177 @@ a Cantor normal form (and injectivity, together with coherence
+      sp sspred reximdva ssrexv sylsyld sylan9r an31s pm2.61dne exlimdv syl6com
+      syld syl5bi imp32 ) BDUAZBDUCZGZCBHZCIJZCDAKZLZIMZACNZWJWICDUAZCDUCZGZWKW
+      OOWJWGWPWHWQCBDUDCBDUMUEWKEKZCPZEUFWRWOECUBWRWTWOEWRWTWOWRWTGWOCDWSLZIWTX
+      AIMZWOOWRWTXBWOWNXBAWSCWLWSMWMXAICDWLWSUGQUHRUIWTWQWPXAIJZWOOWTWQGZWPGXCC
+      DWSUJZCHZXEIJZGZWOXDXCXHOZWPXDXASPZXICDWSUKZXJXCXGXFXJXAXEHZXCXGOCSDWSULX
+      LXCXGXAXEUNRTCSDWSUOZUPTUQWPXHXEDWLLZIMZAXENZXDWOWPFKZCHZXQIJZGZXQDWLLZIM
+      ZAXQNZOZOWPXHXPOZOFXECDWSURXQXEMZYDYEWPYFXTXHYCXPYFXRXFXSXGXQXECUSXQXEIUT
+      VAYBXOAXQXEYFYAXNIXQXEDWLVCQVBVDVEWPYDFVFYDFACDVGYDFVNVHVIXDXFXPWNAXENWOX
+      DXJXFXKXMTZXDXOWNAXEXDWLXEPZGZWNXOYIWMXNIYIXFWMXEHZWMXNMXDXFYHYGUQXDYHYJC
+      DWSWLVJVKCXEDWLVOVLQVMVPWNAXECVQVRVSWDVTWARWBWEWCWF $.
+  $}
+
+  ${
+    $d A y $.  $d B y $.  $d R y $.
+    $( The principle of well-founded induction.  This principle states that if
+       ` B ` is a subclass of a founded class ` A ` with the property that
+       every element of ` B ` whose initial segment is included in ` A ` is
+       itself equal to ` A ` .  Compare ~ wfi and ~ tfi , which are special
+       cases of this theorem that do not require the axiom of infinity to
+       prove.  (Contributed by Scott Fenton, 6-Feb-2011.)  (Revised by Mario
+       Carneiro, 26-Jun-2015.) $)
+    frind $p |- ( ( ( R Fr A /\ R Se A ) /\ ( B C_ A /\
+                       A. y e. A ( Pred ( R , A , y ) C_ B -> y e. B ) ) ) ->
+                   A = B ) $=
+      ( wa wss cpred wcel wn cdif c0 ssdif0 wceq anbi1i cin df-pred incom eqtri
+      wrex bitri wfr wse cv wi wral wne necon3bbii difss frmin eldif anass ccnv
+      ancom csn cima indif2 difeq1i 3eqtr4i eqeq1i bitr4i anbi2i 3bitri rexbii2
+      rexanali sylib ex mpani syl5bi con4d imp adantrl simprl eqssd ) BDUABDUBE
+      ZCBFZBDAUCZGZCFZVPCHZUDABUEZEEBCVNVTBCFZVOVNVTWAVNWAVTWAIBCJZKUFZVNVTIZWA
+      WBKBCLUGVNWBBFZWCWDBCUHVNWEWCEZWDVNWFEWBDVPGZKMZAWBSZWDABWBDUIWIVRVSIZEZA
+      BSWDWHWKAWBBVPWBHZWHEVPBHZWJEZWHEWMWJWHEZEWMWKEWLWNWHVPBCUJNWMWJWHUKWOWKW
+      MWOWHWJEWKWJWHUMWHVRWJWHVQCJZKMVRWGWPKDULVPUNUOZWBOZWQBOZCJWGWPWQBCUPWGWB
+      WQOWRWBDVPPWBWQQRVQWSCVQBWQOWSBDVPPBWQQRUQURUSVQCLUTNTVAVBVCVRVSABVDTVEVF
+      VGVHVIVJVKVNVOVTVLVM $.
+
+    ${
+      frind.1 $e |- R Fr A $.
+      frind.2 $e |- R Se A $.
+      $( The principle of well-founded induction.  This principle states that
+         if ` B ` is a subclass of a founded class ` A ` with the property that
+         every element of ` B ` whose initial segment is included in ` A ` is
+         itself equal to ` A ` .  (Contributed by Scott Fenton, 6-Feb-2011.)
+         (Revised by Mario Carneiro, 26-Jun-2015.) $)
+      frindi $p |- ( ( B C_ A /\
+                         A. y e. A ( Pred ( R , A , y ) C_ B -> y e. B ) ) ->
+                        A = B ) $=
+        ( wfr wse wss cv cpred wcel wi wral wa wceq frind mpanl12 ) BDGBDHCBIBD
+        AJZKCISCLMABNOBCPEFABCDQR $.
+    $}
+  $}
```

Comment:
Is it useful to have this inference, since we have the closed form above ? If a form has to be added, it would rather be the deduction form.

Reply:
These theorems are *old*. They predate the extensive use of deduction form in set.mm. That said, I'm all for eliminating useless theorems. This is now cut.